cookbooks/bcpc-hadoop/attributes/kerberos.rb

```diff
@@ -128,6 +128,15 @@
     perms: '0440',
     spnego_keytab: 'spnego.service.keytab'
   },
+  backup: {
+    principal: node[:bcpc][:hadoop][:backup][:user],
+    keytab: 'backup.service.keytab',
+    owner: node[:bcpc][:hadoop][:backup][:user],
+    group: node[:bcpc][:hadoop][:backup][:user],
+    princhost: '_HOST',
+    perms: '0440',
+    spnego_keytab: 'spnego.service.keytab'
+  },
```

Comment:
This creates a new kerberos keytab.  This process is autmoated on VM clusters, but not in production.  That makes this PR a breaking change.

It should be tagged "breaking" and made a milestone for 4.0